packages/renderer/src/lib/objects/KubernetesUI.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export interface KubernetesUI {
```

Comment:
I think it would be nice to call it `KubernetesObjectUI` to make the link with the `KubernetesObject` provided by the kubernetes library, which is its Kubernetes counterpart:
```
export interface KubernetesObject {
    apiVersion?: string;
    kind?: string;
    metadata?: V1ObjectMeta;
}
```

Another point, for https://github.com/podman-desktop/podman-desktop/issues/11002, I would like to add the `kind` field to this `Kubernetes[Object]UI` interface, so the kind is propagated from the Kubernetes version to the UI version, and we can guess on the UI version of the object what is its kind.